finetune/base_models/gpt/encoder.py

```diff
@@ -179,7 +177,7 @@ def _encode(self, texts, labels=None):
                     token
                 ):  # the BPEs comprising a token are longer than the token itself
                     token_char_ends = (
-                        np.asarray([len(token.text.strip()) for tok in bpe_toks])
+                        np.asarray([len(token.text.strip()) for _ in bpe_toks])
```

Comment:
Seems like this sets every subtokens ends to the full token end?

Reply:
Yes, I don't like it or understand it. I removed the equivalent from gpt2 and can probably do the same thing here.